File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Identification.java
package com.babestudios.companyinfouk.data.model.persons;

import com.google.gson.annotations.SerializedName;

public class Identification {
	@SerializedName("country_registered")
	public String countryRegistered;
	@SerializedName("legal_authority")
	public String legalAuthority;
	@SerializedName("legal_form")
	public String LegalForm;
	@SerializedName("place_registered")
	public String placeRegistered;
	@SerializedName("registration_number")
	public String registrationNumber;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Identification.kt
package com.babestudios.companyinfouk.data.model.persons

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Identification(
		@SerializedName("country_registered")
		var countryRegistered: String? = null,
		@SerializedName("legal_authority")
		var legalAuthority: String? = null,
		@SerializedName("legal_form")
		var LegalForm: String? = null,
		@SerializedName("place_registered")
		var placeRegistered: String? = null,
		@SerializedName("registration_number")
		var registrationNumber: String? = null
) : Parcelable
